index.js

```diff
@@ -114,6 +114,16 @@ function SendStream(req, path, options) {
     ? normalizeList(opts.extensions, 'extensions option')
     : []
 
+  if (Array.isArray(opts.precompressed)) {
+    this._precompressionFormats = opts.precompressed
+  } else if (opts.precompressed) {
+    this._precompressionFormats = [{encoding: 'br', extension:'.br'}, {encoding:'gzip', extension:'.gz'}]
```

Comment:
If, by default, we are going to search for files with the `.br` extension, I want to be sure this is actually the official file extension for this format and it does not become something else at some point. Can you point me to somewhere that says the file extension for brotli (ideally some place with authority in the brotli project). All I can find is the `Content-Encoding` defined as `br`, but no mention of a file extension.